PocketKit/Sources/Sync/ApolloClientProtocol+Extensions.swift

```diff
@@ -80,7 +80,8 @@ public extension ApolloClientProtocol {
         let skippableErrors = [403]
 
         guard let responseError = error as? ResponseCodeInterceptor.ResponseCodeError else {
-            Log.capture(message: "GraphQl Error - unknown error.", filename: filename, line: line, column: column, funcName: funcName)
+
+            Log.capture(message: "GraphQl Error - unknown error: \(error.localizedDescription)", filename: filename, line: line, column: column, funcName: funcName)
```

Comment:
adding error info is absolutely valid. One thing I'd change, based on my experience is using `\(error)` instead of `\(error.localizedDescription)` because the former usually carries a lot more info, while the latter will often contain messages like "unknown error" or such.